vmsdk/python/tests/inc/platform.py

```diff
@@ -0,0 +1,16 @@
+"""Platform specific check factory."""
+from cctrusted_vm.cvm import ConfidentialVM
+from cctrusted_vm.sdk import CCTrustedVmSdk
+from tests.tdx.tdx_check import CCTrustedVmSdkTdxCheck
+
+class PlatformCheck():

```

Comment:
It looks wierd to me for this kind class

Reply:
The motivation of adding this is to get some mechanism so I can add some platform specific test: for example, Intel TDX TDREPORT provides the 4 measurement registers.

I'm sorry that I haven't figure out a good way to do so. But I'll try to find out a better way.